src/families/crypto_org/js-getFeesForTransaction.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import { FIXED_GAS_PRICE, FIXED_DEFAULT_GAS_LIMIT } from "./logic";
+
+/**
+ * Fetch the transaction fees for a transaction
+ */
+const getEstimatedFees = async (): Promise<BigNumber> => {
+  // Todo call gas station to get a more accurate tx fee in the future
```

Comment:
fixed. It is not something that need to be fixed because it practice, it should work. The TODO is more like an optimisation if we ever have dynamic gas fee in the future